Neos.Flow/Classes/ResourceManagement/ResourceManager.php

```diff
@@ -218,8 +233,7 @@ public function importResourceFromContent($content, $filename, $collectionName =
             throw new Exception(sprintf('Importing content into the resource collection "%s" failed: %s', $collectionName, $exception->getMessage()), 1381156155, $exception);
         }
 
-        $this->resourceRepository->add($resource);
-        $this->logger->debug(sprintf('Successfully imported content into the resource collection "%s" (storage: %s, a %s. SHA1: %s)', $collectionName, $collection->getStorage()->getName(), get_class($collection->getStorage()), $resource->getSha1()));
+        $this->persistResource($resource, $collection, 'content');
```

Comment:
We could improve the source hint here to the `$filename`.

Reply:
That'd be nice.

Reply:
```suggestion
        $this->persistResource($resource, $collection, sprintf('content as "%s"', $filename));
```